function-calling/multi-tool/create_athena_catalog.py

```diff
@@ -0,0 +1,76 @@
+import boto3
+
+bucket = 'bucketname' ### REPLACE WITH YOUR AMAZON S3 BUCKET
+
+### Setup boto3 client for Athena
+athena = boto3.client('athena', region_name='us-east-1')
+
+### Create the Athena database
+athena.start_query_execution(
+    QueryString='CREATE DATABASE IF NOT EXISTS acme_bank',
+    ResultConfiguration={
+        'OutputLocation': f's3://{bucket}/athena/'
+    }
+)
+
+### Create the transactions table
+athena.start_query_execution(
+    QueryString=f'''
+        CREATE EXTERNAL TABLE IF NOT EXISTS acme_bank.transactions (
+            user_id INT,
+            user_name STRING,
+            transaction_date DATE,
+            amount DECIMAL(10,2)
+        )
+        ROW FORMAT DELIMITED
+        FIELDS TERMINATED BY ','
+        STORED AS TEXTFILE
+        LOCATION 's3://{bucket}/acme/'
+    ''',
+    ResultConfiguration={
+        'OutputLocation': f's3://{bucket}/athena/'
+    }
+)
+
+### Insert synthetic example data
+transactions_data = [
+    (1, 'Tom Hanks', '2023-01-01', 500.00),
```

Comment:
is this data generated? do we have permissions to use it?

Reply:
I found it from https://github.com/aws-samples/amazon-bedrock-samples/blob/main/function-calling/create_athena_catalog.py